modules/services/avizo.nix

```
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkEnableOption ''
+      avizo, a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway
+    '';
```

Either split the line to make it a bit shorter or just simplify the description :-)

```suggestion
    enable = mkEnableOption "avizo, a simple notification daemon";
```

If you want to keep the longer description make sure you don't have a newline at the end of the text. Otherwise there will be a space between the last character and the period.